bundles/io/org.eclipse.smarthome.io.rest.sitemap/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+	http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="rest:sitemap">
```

Comment:
Such settings are usually done under the "system" category - so this could be "system:sitemapsubscription" (compare e.g. with `system:chart`).

Reply:
On the other hand, I am wondering whether we want this at all to turn up in the UI - it is very technical and should normally never touched. Being able to set it through a config file would imho suffice.

Reply:
I agree that this configuration is not for the "user" but more for the system admin or the product maintainer. For me it does not need to be present in the UI.

Reply:
What shoild I change to have the setting available through file but not UI ?